Server/VikingVault.DataAccess/Models/AutomaticPayment.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace VikingVault.DataAccess.Models
+{
+    public class AutomaticPayment
+    {
+        public int Id { get; set; }
+        [Required]
+        public User Company { get; set; }
```

Comment:
Small thing, but it would make more sense to name this "RecievingCompany". Naming database columns descriptively is hugely important to understand how the application works.